nucypher/blockchain/eth/actors.py

```diff
@@ -535,7 +532,7 @@ def __init__(self, filepath: str, registry, deployer_address):
         self.economics = EconomicsFactory.get_economics(registry)
 
         self.__total_to_allocate = 0
-        self.__process_allocation_data(filepath)
+        self.__process_allocation_data(str(filepath))
```

Comment:
Is `str(...)` necessary here? The typehint for filepath already suggests it is a string.

Reply:
I was just having a bit of troube with needing to cast a `Path`.